todo_app/data/trello_items.py

```diff
@@ -0,0 +1,80 @@
+from flask import session
+
+from todo_app.data.Item import Item
+
+import os, requests
+
+def get_items():
+    """
+    Fetches all saved items from the session.
+
+    Returns:
+        list: The list of saved items.
+    """
+
+    BOARD_ID = os.getenv('CORNDEL_BOARD_ID')
+    API = os.getenv('TRELLO_API')
+    BOARD = os.getenv('TRELLO_BOARD').replace('{id}', BOARD_ID)
+    request_url = API+BOARD+'?key='+os.getenv('TRELLO_API_KEY')+'&token='+os.getenv('TRELLO_TOKEN')
```

Comment:
I had to spend a few minutes tracking down all of these environment variables, understanding what they were, checking the Trello docs, and then adding them to my `.env` file so I could run your app - it would be nice to collect those in one place (in your `.env.template` file!) to make that a bit easier for future devs, e.g. we could add:

```
TRELLO_API=https://api.trello.com/1
TRELLO_BOARD=/boards/{id}/cards
TRELLO_CARDs=/cards
TRELLO_CARD=/cards/{id}

CORNDEL_BOARD_ID=add-your-board-id
CORNDEL_TODO_LIST_ID=add-your-todo-list-id
CORNDEL_DONE_LIST_ID=add-your-done-list-id

TRELLO_API_KEY=add-your-key
TRELLO_TOKEN=add-your-token
```

Reply:
Note that the fact some of these don't really need to be provided by the user (e.g. the API isn't likely to move!) might suggest we'd be better off making that a variable in the code, rather than an environment variable, but the cost is very minor